macosx/TorrentCell.mm

```diff
@@ -107,7 +107,7 @@ - (instancetype)init
         _fTitleAttributes[NSParagraphStyleAttributeName] = paragraphStyle;
 
         _fStatusAttributes = [[NSMutableDictionary alloc] initWithCapacity:3];
-        _fStatusAttributes[NSFontAttributeName] = [NSFont messageFontOfSize:9.0];
+        _fStatusAttributes[NSFontAttributeName] = [NSFont messageFontOfSize:9.5];
```

Comment:
Both the original 9 and this modified 9.5 seem strange to me. it looks like we are hardcoding font sizes in about four places here which seems odd to me. What is the default point size -- 10? -- such that the upshot of this line is "make it slightly smaller than the default"?

I'd be more comfortable with something that reacts cleanly to changes in the system default fonts, e.g. by making this relative to the default size --

```
CGFloat const default_message_point_size = [[NSFont messageFontOfSIze:-1] pointSize];
_fStatusAttributes[NSFontAttributeName] = [NSFont messageFontOfSize:default_message_point_size * 0.9];
```

(I have no idea if 90% is the desirable percentage; it's only here as an example)

Reply:
Yes, we have variables in both the xibs and the mm's. This is a legibility fix, if i increase to 10, then other attributes should be raised accordingly, and i didn't want to get into that until after 4.00. That .5 makes a big difference on my monitor.